coldfront/core/user/views.py

```diff
@@ -309,6 +310,104 @@ def test_func(self):
         return self.request.user.is_staff
 
 
+class UserSearchAll(LoginRequiredMixin, ListView):
+    model = User
+    template_name = 'user/user_list.html'
+    context_object_name = 'user_list'
+    paginate_by = 25
+
+    def test_func(self):
+        return self.request.user.is_staff
```

Comment:
This should be `is_superuser`. We will open up permissions later once we've reviewed who should get access to what.